cpg-language-python/src/main/kotlin/de/fraunhofer/aisec/cpg/frontends/python/PythonLanguage.kt

```diff
@@ -42,6 +42,8 @@ class PythonLanguage : Language<PythonLanguageFrontend>(), HasShortCircuitOperat
     override val frontend: KClass<out PythonLanguageFrontend> = PythonLanguageFrontend::class
     override val conjunctiveOperators = listOf("and")
     override val disjunctiveOperators = listOf("or")
+    override val compoundAssignmentOperators =
```

Comment:
Maybe ref https://docs.python.org/3/library/operator.html#in-place-operators. This was the best place that I have found.

It seems `@=` is missing (new in Python 3.5)